pkg/registry/models/util.go

```diff
@@ -251,20 +251,25 @@ func validateModelDir(dirPath string, model *Model) error {
 	if err != nil {
 		return err
 	}
-	for _, file := range fileList {
-		err = util.ExecOSCommand("mv",
-			[]string{
-				path.Join(fileListPath, file.Name()),
-				ormbModelDir})
 
+	// if `fileListPath != ormbModelDir` logic is special for issue
```

Comment:
Why can this PR fix the problem when the zip name is model?

Reply:
because the bug is that source dir is same as destination dir when execute mv command. so if the dir name is same, not execute mv command.